Automattic/wp-calypso
# Jetpack checklist: remove jetpack/checklist/performance flag and un-u…

Since the performance checklist is stable, we can remove the flag.

#### Changes proposed in this Pull Request

* Remove`jetpack/checklist/performance` flag and un-used code that was shown only when the flag was disabled.

#### Testing instructions

- Connect Jetpack, choose a plan _or_ have an existing jetpack site with plan
- Open `/plans/my-plan/SITE`
- Confirm that lazy loading images, site accelerator, video hosting and search -performance checklist items behave as expected and you don't see their feature cards under the checklist.
